README.md

```diff
@@ -58,8 +59,10 @@ repositories {
 dependencies {
     // Other dependencies
     compile group: 'com.cloudant', name: 'cloudant-sync-datastore-core', version:'0.12.1'
-    // include this if you're targeting Android
+    // include this if you're targeting Android and do not want datastore encryption
     compile group: 'com.cloudant', name: 'cloudant-sync-datastore-android', version:'0.12.1'
+    // include this if you're targeting Android and want datastore encryption
+    compile group: 'com.cloudant', name: 'cloudant-sync-datastore-android-encryption', version:'0.12.1'
```

Comment:
To clarify my understanding, what we're saying here is that the `-encryption` jar is a substitute rather than in addition to the `cloudant-sync-datastore-android` jar?